qt_ui/windows/QDebriefingWindow.py

```diff
@@ -86,3 +88,9 @@ def __init__(self, debriefing: Debriefing):
         okay = QPushButton("Okay")
         okay.clicked.connect(self.close)
         layout.addWidget(okay)
+
+    def closeEvent(self, event: QCloseEvent) -> None:
+        super().closeEvent(event)
+        state = self.debriefing.game.check_win_loss()
+        if state == TurnState.WIN or state == TurnState.LOSS:
+            GameUpdateSignal.get_instance().endGame(state)
```

Comment:
Upon closing of the debriefing window, check for win/loss and emit signal if required.